unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/RPC/GeneratedCode/RendererProtocolService.gen.cs

```diff
@@ -0,0 +1,44 @@
+// AUTOGENERATED, DO NOT EDIT
+// Type definitions for server implementations of ports.
+// package: 
+// file: RendererProtocol.proto
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using Google.Protobuf;
+using rpc_csharp.protocol;
+using rpc_csharp;
+
+public abstract class CRDTService<Context>
+{
+    public const string ServiceName = "CRDTService";
+
+    public delegate UniTask<CRDTResponse> SendCrdt(CRDTManyMessages request, Context context);
+
+    public delegate IEnumerator<CRDTManyMessages> CrdtNotificationStream(CRDTStreamRequest request, Context context);
+
+    public static void RegisterService(RpcServerPort<Context> port, SendCrdt sendCrdt, CrdtNotificationStream crdtNotificationStream)
+    {
+        var result = new ServerModuleDefinition<Context>();
+      
+        result.definition.Add("SendCrdt", async (payload, context) => { var res = await sendCrdt(CRDTManyMessages.Parser.ParseFrom(payload), context); return res?.ToByteString(); });
+        result.streamDefinition.Add("CrdtNotificationStream", (payload, context) => { return new ProtocolHelpers.StreamEnumerator<CRDTManyMessages>(crdtNotificationStream(CRDTStreamRequest.Parser.ParseFrom(payload), context)); });
+
+        port.RegisterModule(ServiceName, (port) => UniTask.FromResult(result));
+    }
+}
+
+public abstract class PingPongService<Context>
```

Comment:
Do you want to keep with this test Service? 🤔

Reply:
it should be removed after merging this PR, right? https://github.com/decentraland/unity-renderer/pull/2577